src/stores/sagas/orderLineDetailsSaga.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  all, call, put, takeLatest,
+} from 'redux-saga/effects';
+
+import { IOrderLineDetailsType } from 'models/orderLineDetails/IOrderLinedetailsType';
+import {
+  getOrderLineDetailsFailure,
+  getOrderLineDetailsSuccess,
+  postOrderLineDetailsFailure,
+  postOrderLineDetailsSuccess,
+} from 'stores/actions/orderLineDetails/orderLineDetailsActions';
+import OrderLineDetailsActionTypes from 'stores/actions/orderLineDetails/orderLineDetailsTypes';
+import { sessionToken } from 'stores/reducers/tokenReducer';
+import refreshAccessToken from 'stores/sagas/utils';
+import apiClient, { getRequest } from 'utils/apiClient';
+
+export const orderLineDetailsUrl = 'order_line_details/';
+
+const postRequest = async (payload: object, id: number) => {
```

Comment:
It is good that you decided here to overwrite postRequest method in that context!